contributor-growth/drafts/contributor-growth-framework/pr-worflow.md

```diff
@@ -0,0 +1,54 @@
+# 2. A successful PR workflow  
+
+The PR workflow is at the core of code contributions. Make sure it's a smooth experience and try to get contributors to contribute something else once they've succeeded. 
+
+Here's a typical PR workflow: 
+   1. **User opens an issue** (bug, feature request, vulnerability): As maintainers, you will first troubleshoot the bug or review the feature request. Reach out to the submitter to thank them for the issue and assess if they want to be involved. If they are, ask them to continue the conversation on your contributor channel. 
+
+   2. **Pick conversation up on contributor channel**:  
+Projects should consider assigning new contributors to a mentor or point of contact who can follow up making sure they remain motivated and follow through. Community CRMs such as [Orbit](https://orbit.love/) or [Savannah](https://www.savannahhq.com/) will help you keep track of contributor activity. If you don't see any activity from your prospective contributor, check-in to see if they are still interested. A gentle reminder may revive their motivation and show them you really care about their contribution. 
```

Comment:
Did anyone suggest how they track interactions using free tooling? I'm not terribly comfortable recommending these so strongly and often in the docs.

I haven't ever worked on a project that _does_ use a CRM tool actually. I think it would be better to say these exist and how people get by with tools provided by GitHub.